<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerBreachWarningModule</class>
 <widget class="qSlicerWidget" name="qSlicerBreachWarningModule">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Breach warning</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Parameter set:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="ParameterNodeComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLBreachWarningNode</string>
        </stringlist>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
       <property name="selectNodeUponCreation">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Model to watch:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="qMRMLNodeComboBox" name="ModelNodeComboBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="nodeTypes">
             <stringlist>
              <string>vtkMRMLModelNode</string>
             </stringlist>
            </property>
            <property name="noneEnabled">
             <bool>true</bool>
            </property>
            <property name="addEnabled">
             <bool>false</bool>
            </property>
            <property name="removeEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Tool tip (to RAS) transform: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="qMRMLNodeComboBox" name="ToolComboBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="nodeTypes">
             <stringlist>
              <string>vtkMRMLLinearTransformNode</string>
             </stringlist>
            </property>
            <property name="noneEnabled">
             <bool>true</bool>
            </property>
            <property name="addEnabled">
             <bool>false</bool>
            </property>
            <property name="removeEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>185</height>
      </size>
     </property>
     <property name="text">
      <string>Parameters</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="LineToClosestPointVisibilityCheckBox">
        <property name="text">
         <string>Display line to closest point</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkColorPickerButton" name="LineToClosestPointColorPickerButton">
        <property name="color">
         <color>
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </property>
        <property name="colorName">
         <string>Line to closest point color</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="WarningCheckBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Display warning color</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ctkColorPickerButton" name="WarningColorPickerButton">
        <property name="color">
         <color>
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </property>
        <property name="colorName">
         <string>Warning color</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="SoundCheckBox">
        <property name="text">
         <string>Play warning sound</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="LineToClosestPointTextSizeLabel">
        <property name="text">
         <string>Line to closest point text size:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ctkSliderWidget" name="LineToClosestPointTextSizeSlider">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="LineToClosestPointThicknessLabel">
        <property name="text">
         <string>Line to closest point thickness:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="ctkSliderWidget" name="LineToClosestPointThicknessSlider">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="WarningDistanceMMLabel">
        <property name="text">
         <string>Breach warning distance (mm):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkSliderWidget" name="WarningDistanceMMSlider">
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerBreachWarningModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ModelNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerBreachWarningModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ParameterNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerBreachWarningModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ToolComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
